virttest/utils_net.py

```diff
@@ -1215,12 +1215,12 @@ def setup_ovs_vhostuser(hp_num, tmpdir, br_name, port_names,
                   clean_ovs=True)
 
     # Install openvswitch
-    if process.system("yum info openvswitch", ignore_status=True) == 0:
-        utils_package.package_install("openvswitch")
-    if process.system("yum info openvswitch2.11", ignore_status=True) == 0:
-        utils_package.package_install("openvswitch2.11")
     if process.system("yum info openvswitch2.15", ignore_status=True) == 0:
```

Comment:
it seems that wrap them into one loop looks more straightforward:
```
for ovs_package in [openvswitch2.15,openvswitch2.11, openvswitch]:
       if process.system("yum info %s" % ovs_package , ignore_status=True) == 0:
              utils_package.package_install(ovs_package)
               break
```

Reply:
Thanks! updated.